app.py

```diff
@@ -69,7 +63,7 @@ def detect():
         
     IMG_TAIL = random.randint(0, 1000000000)
     file.save(os.path.join(UPLOAD_FOLDER, str(IMG_TAIL) + 'result.png'))
-    return jsonify(response['FaceDetails'], str(IMG_TAIL) + 'result.png')
+    return jsonify(response['FaceDetails'], 'image.png')
```

Comment:
I think this `file.save(os.path.join(UPLOAD_FOLDER, str(IMG_TAIL) + 'result.png'))` should be replaced by `file.save(os.path.join(UPLOAD_FOLDER, 'image.png'))`. 
Isn't it ?

Reply:
Yes

Reply:
Can you fix ? Then I will merge this pull request.

Reply:
You can merge them into your repo and update the changes. As I've already dumped repo from my local system.

Reply:
Okay. Thank you for the contribution.